src/billing/components/marketplace/MarketplaceLink.tsx

```diff
@@ -6,43 +6,32 @@ import {
   ButtonShape,
 } from '@influxdata/clockface'
 
-import {Marketplace} from 'src/types/marketplace'
+// Types
+import {Marketplace} from 'src/types/billing'
 
 // Utils
 import {useBilling} from 'src/billing/components/BillingPage'
 
-const buttonInfo = (
-  marketplace: Marketplace['shortName']
-): {link: string; text: string} => {
+const buttonInfo = (marketplace: Marketplace['shortName']): string => {
   switch (marketplace) {
     case 'aws':
-      return {
-        link: 'https://aws.amazon.com/marketplace/library/',
-        text: 'AWS Subscription Portal',
-      }
+      return 'AWS Subscription Portal'
```

Comment:
nit: marketplace will have a `name` field on it that should be respectively `AWS`, `Google Cloud` and `Microsoft Azure`. Not sure if its worth using those here, but just a thought.